docs/extensions.adoc

```diff
@@ -525,7 +525,8 @@ def ignoreMyExceptions
 === TempDir
 
 In order to generate a temporary directory for test and delete it after test, annotate a member field of type
-`java.io.File`, `java.nio.file.Path` or untyped using `def` in a spec class (`def` will inject a `Path`). If the
+`java.io.File`, `java.nio.file.Path` or untyped using `def` in a spec class (`def` will inject a `Path`).
+Alternatively, you can annotate a field with a custom type that has a public constructor accepting either `java.io.File` or `java.nio.file.Path` as its single parameter (see <<utilities.adoc#fs-fixture, FsFixture>> as an example). If the
```

Comment:
Just because it sounds a little nicer:

```suggestion
Alternatively, you can annotate a field with a custom type that has a public constructor accepting either `java.io.File` or `java.nio.file.Path` as its single parameter (see <<utilities.adoc#fs-fixture, FsFixture>> for an example). If the
```